Mixpanel/Mixpanel.m

```diff
@@ -1029,9 +1029,11 @@ - (NSDictionary *)collectAutomaticProperties
                                   @"mp_lib": @"iphone",
                                   @"$lib_version": [self libVersion],
                                   @"$manufacturer": @"Apple",
-                                  @"$model": deviceModel,
```

Comment:
i guess that's fine! I wonder if you could also do something like: `deviceModel ?: [NSNull null]`